syntax = "proto3";
message MP3DownloadRequest {
    string collection_id = 1;  
    string episode_id = 2;
    optional string episode_url = 3; // support seach against firestore
}